extensions/amp-form/0.1/amp-form-textarea.js

```diff
@@ -77,7 +77,7 @@ export class AmpFormTextarea {
     this.doc_ = (root.ownerDocument || root);
 
     /** @private @const */
-    this.win_ = devAssert(this.doc_.defaultView);
+    this.win_ = /** @type {!Window} */ (devAssert(this.doc_.defaultView));
```

Comment:
all the dev asserts in this file are pretty pointless and can be removed.

Reply:
Unfortunately, they're needed. Without them:
```js
extensions/amp-form/0.1/amp-form-textarea.js:116: ERROR - actual parameter 1 of throttle$$module$src$utils$rate_limit does not match formal parameter
found   : (Window|null)
required: Window
    const throttledResize = throttle(this.win_, e => {
                                     ^^^^^^^^^

extensions/amp-form/0.1/amp-form-textarea.js:233: ERROR - actual parameter 1 of computedStyle$$module$src$style does not match formal parameter
found   : (Window|null)
required: Window
    const computed = computedStyle(win, element);
                                   ^^^

extensions/amp-form/0.1/amp-form-textarea.js:290: ERROR - actual parameter 1 of Resources$$module$src$service$resources_impl.prototype.measureMutateElement does not match formal parameter
found   : (HTMLBodyElement|null)
required: Element
  return resources.measureMutateElement(body, () => {
                                        ^^^^

extensions/amp-form/0.1/amp-form-textarea.js:291: ERROR - actual parameter 1 of computedStyle$$module$src$style does not match formal parameter
found   : (Window|null)
required: Window
    const computed = computedStyle(win, textarea);
                                   ^^^

extensions/amp-form/0.1/amp-form-textarea.js:305: ERROR - actual parameter 1 of Resources$$module$src$service$resources_impl.prototype.measureMutateElement does not match formal parameter
found   : (HTMLBodyElement|null)
required: Element
    return resources.measureMutateElement(body, () => {
                                          ^^^^
```